get_git.go

```diff
@@ -180,17 +180,34 @@ func (g *GitGetter) fetchSubmodules(dst, sshKeyFile string) error {
 // setupGitEnv sets up the environment for the given command. This is used to
 // pass configuration data to git and ssh and enables advanced cloning methods.
 func setupGitEnv(cmd *exec.Cmd, sshKeyFile string) {
-	var sshOpts []string
+	gitSSHCommand := "GIT_SSH_COMMAND="
```

Comment:
Nitpick I'd probably just do `const gitSSHCommand =` here since we never use this as a var. But it has no real functional effect here.